nucliadb_dataset/nucliadb_dataset/settings.py

```diff
@@ -19,13 +19,14 @@
 #
 
 from pathlib import Path
-from typing import List, Optional
+from typing import Optional
 
 import pydantic
 from pydantic import BaseSettings
 
-from nucliadb_dataset import DatasetType, ExportType
-from nucliadb_sdk.client import Environment
+from nucliadb_dataset import ExportType
+from nucliadb_dataset.dataset import Task
```

Comment:
`Task` is defined in `tasks` module, not in `dataset`

```suggestion
from nucliadb_dataset.tasks import Task
```